src/main/webapp/app/exercises/shared/participation-submission/participation-submission.component.html

```diff
@@ -39,20 +39,20 @@
                 {{ value | artemisDate: 'long':true }}
             </ng-template>
         </ngx-datatable-column>
-        <ngx-datatable-column prop="result">
+        <ngx-datatable-column prop="results">
             <ng-template ngx-datatable-header-template>
                 <span jhiTranslate="artemisApp.participation.participationSubmission.result"></span>
             </ng-template>
             <ng-template ngx-datatable-cell-template let-value="value">
-                <jhi-result [result]="value" [participation]="participation"></jhi-result>
+                <jhi-result [result]="value && value.length > 0 ? value[value.length - 1] : null" [participation]="participation"></jhi-result>
```

Comment:
please use `undefined` instead of `null`

Reply:
another (preferred) option would be to make sure `latestResult` is initialized correctly (see #2551) in `submission` which should be done in the `submission.service.ts` within `findAllSubmissionsOfParticipation`. Then we could simply use `latestResult` as `prop` in line 42 and keep using `value` in line 47
@nicolasruscher could